itest/lnd_channel_force_close_test.go

```diff
@@ -6,7 +6,6 @@ import (
 
 	"github.com/btcsuite/btcd/btcutil"
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
-	"github.com/btcsuite/btcd/wire"
```

Comment:
suggestion in commit message: split up force close tests

Reply:
updated